src/vos/tests/evt_ctl.c

```diff
@@ -939,7 +946,9 @@ test_evt_iter_flags(void **state)
 	rc = evt_create(EVT_FEAT_DEFAULT, 13, arg->ta_uma, arg->ta_root,
 			DAOS_HDL_INVAL, &toh);
 	assert_int_equal(rc, 0);
-	data = (int **)malloc((NUM_EPOCHS+1)*sizeof(int *));
+	D_ALLOC_ARRAY(data, ((NUM_EPOCHS+1)*sizeof(int *)));
```

Comment:
D_ALLOC_ARRAY uses the size of the entity the pointer points at

So, the sizeof(int *) is not necessary here.  Just allocate using the number of entries in the array.

So this is actually equivalent to a calloc of (NUM_EPOCHS+1)*sizeof(int *)*sizeof(*data) which is too large.

Reply:
Done